<?xml version="1.0"?>

<launch>
    <node pkg="local_planner" type="local_planner_debugger" name="local_planner_debugger">
        <param name="debug_mode" type="int" value="1"/>
    </node>
</launch>
    
